system/Test/CIUnitTestCase.php

```diff
@@ -125,7 +126,7 @@ protected function createApplication()
 		$systemPath = realpath(__DIR__.'/../');
 
 		require_once $systemPath.'/'.$this->configPath.'/Paths.php';
-		$paths = $this->adjustPaths(new \Config\Paths());
+		$this->adjustPaths(new Paths());
```

Comment:
```suggestion
		$paths = $this->adjustPaths(new Paths());
```

Removing that return value is why you're seeing the error you're seeing on Travis.

Reply:
I've found that PHPStorm occasionally lies about random things. :)